package.json

```diff
@@ -14,17 +14,17 @@
     "cookie-parser": "1.4.3",
     "express": "4.14.0",
     "express-session": "1.14.2",
-    "infusion": "2.0.0-dev.20161027T155641Z.0bcd507",
-    "resolve": "1.1.7"
+    "infusion": "2.0.0-dev.20161219T170555Z.5778f7e",
+    "resolve": "1.2.0"
```

Comment:
Good catch.  I reviewed in depth, the dev "request" dependency is also unused.  Both have been removed.